MonopolyGame/src/cs414/a4/monopoly/backEnd/Board.java

```diff
@@ -44,6 +44,34 @@ public Board(List<Player> players) {
 	public List<Player> getPlayers(){
 		return players;
 	}
+	
+	public Property movePlayer(Player player, int distance){
+		player.movePlayer(distance);
+		return boardSpaces[player.getLocation()].getDeed();
+	}
+	
+	public void sellPropertyFromBankToPlayer(Property deed, Player player){
+		player.propertiesOwned.add(deed);

```

Comment:
I believe all the lines player.propertiesOwned.add(deed) should be calling player.addProperty(property, cost) function. This checks to make sure the player can afford it. I believe the lines i say need replaced are directly manipulating the list and would need to have a accessor function. maybe discuss this with the team.